fe1-web/model/objects/Base64Url.ts

```diff
@@ -8,8 +8,8 @@ export class Base64UrlData extends String {
     base64url.decode(value);
   }
 
-  public static encode(text: string | Buffer) : Base64UrlData {
-    return new Base64UrlData(this.addPadding(base64url.encode(text)));
+  public static encode(text: string | Buffer, encoding: string = 'utf8') : Base64UrlData {
+    return new Base64UrlData(this.addPadding(base64url.encode(text, encoding)));
```

Comment:
the `base64url.encode` by default assumes that the string is utf8 encoded.
It's perfect if you're hashing a string, not so much a binary array.